tutor/src/routes.js

```diff
@@ -9,6 +9,9 @@ const getRoutes = (router) => {
     const ConditionalHandlers = getConditionalHandlers(router);
 
     return [
+        { path: '/dashboard', name: 'oldMyCourses',
```

Comment:
I think the reload sends the user to `/courses`

Reply:
it does on the current code, but the code that's already on production uses /courses, and that's what users will be running when this is deployed

Reply:
👍